ocw-cli/cli_app.py

```diff
@@ -1138,30 +1080,36 @@ def settings_screen(header):
               new_start_time = screen.getstr()
               try:
                    new_start_time = datetime.strptime(new_start_time, '%Y-%m-%d')
-                   new_start_time_int = int("{0}{1}".format(new_start_time.year, new_start_time.month))
-                   fix_min_time_int = int("{0}{1}".format(fix_min_time.year, fix_min_time.month))
-                   fix_max_time_int = int("{0}{1}".format(fix_max_time.year, fix_max_time.month))
-                   all_overlap_end_time_int = int("{0}{1}".format(all_overlap_end_time.year, all_overlap_end_time.month))
-                   if new_start_time_int < fix_min_time_int \
-                   or new_start_time_int > fix_max_time_int \
-                   or new_start_time_int > all_overlap_end_time_int:
-                        note = "Start time has not changed. "
+                   #new_start_time_int = int("{0}{1}".format(new_start_time.year, new_start_time.month))
```

Comment:
If we're going to remove this code don't comment it out, just remove it. There's no reason to leave commented out code in the code base.